ArduPlane/reverse_thrust.cpp

```diff
@@ -98,7 +98,7 @@ bool Plane::allow_reverse_thrust(void) const
     }
 
     // cope with bitwise ops above
-    return allow != false;
+    return allow;
```

Comment:
Well, the comment is now confusing :-)

It should always have been `!= 0` IMO.  Given your citing of the standard and testing of 4.9 this is probably OK once the comment is removed.